scripts/Diffraction/isis_powder/abstract_inst.py

```diff
@@ -86,6 +91,16 @@ def _focus(
             paalman_pings_events_per_point=paalman_pings_events_per_point,
         )
 
+    def _output_focused_runs(self, focused_runs, run_number_string):
+        run_details = self._get_run_details(run_number_string)
+        for focused_run in focused_runs:
```

Comment:
If I look at the original focus.py code where this used to live, there wasn't a for loop corresponding to this. Is the loop required?

Reply:
As discussed - you have moved the file output so it is outside the loop in `_individual_run_focusing` in focus.py. So fine to have this for loop, I understand now